toonz/sources/toonz/main.cpp

```diff
@@ -385,7 +385,7 @@ int main(int argc, char *argv[]) {
   fmt.setStencil(true);
   QGLFormat::setDefaultFormat(fmt);
 
-#ifdef LINUX
+#if defined(LINUX) || (defined(_WIN32) && defined(__GNUC__))
```

Comment:
While this is a  small example, why would function call on MINGW not be called with MSVC ?

Having MSVC & MINGW behave differently WRT function calls seems like its going to cause mismatch on behavior, possibly bugs on one system but not others.